### ISSUE TITLE
User Filer Settings - sort, bulk select, bulk set

### ISSUE THREAD

**Issue by mkocikowski** [EFForg/privacybadger#691]
Privacy Badger Options --> User Filter Settings tab should have the ability to sort tracking domains (by color), and to bulk select / set domains ("set all green to red").

**Comment by cooperq** [comment]
I disagree, I think that such features would only serve to clutter the UI and slow down the page. More options is almost never a good decision for UI. And giving users the ability to change domain settings in bulk will likely lead to people making mistakes and unblocking or blocking things they didn't plan to. 
I will not implement this feature.

**Comment by mkocikowski** [comment]
I currently show 565 "potential tracking domains". The current UI displays only 7 of them at a time, sorted alphabetically. Without the ability to sort or search the list, it is, for me, very difficult to interact with this data. Is there a way to access this data through other means (other than the graphical UI)?

**Comment by syeo66** [comment]
At least it would be great to see all the manually adjusted domains without having to skim through all the 993 entries in my current list. This should be the absolute minimum... without that Privacy Badger will soon be worthless for me.

**Comment by cooperq** [comment]
Your point is well taken. I do agree that searching would at least be nice. And I can see how sorting user overrides to the top would be useful for some. I'm still not comfortable with bulk select / bulk set but I think that some amount of searching and sorting is useful. Removing this from wontfix.

**Comment by KidPluto** [comment]
This one looks like it has been completed.  Thanks.
